os/board/rtl8721csm/src/component/os/tizenrt/rtk_netmgr.c

```diff
@@ -83,15 +83,86 @@ static wifi_utils_scan_list_s *g_scan_list;
 static int g_scan_num;
 extern struct netdev *ameba_nm_dev_wlan0;
 
+struct ap_scan_list {
+	unsigned int channel;
+	char ssid[WIFI_UTILS_SSID_LEN + 1];
+	unsigned int ssid_length;
+	int rssi;
+	wifi_utils_ap_auth_type_e ap_auth_type;
+	wifi_utils_ap_crypto_type_e ap_crypto_type;
+};
+typedef struct ap_scan_list ap_scan_list_s;
+
+static ap_scan_list_s *saved_scan_list;
+static uint32_t scan_number = 0;
+uint32_t ap_channel;
+
+#include "freertos/wrapper.h"
+#define SCAN_TIMER_DURATION 180000
+_timer scan_timer;

```

Comment:
If this variable is used only in this file, please add `static`.

Reply:
Done.